plugins/bulk-action.yaml

```diff
@@ -21,11 +21,14 @@ spec:
   shortDescription: Do bulk actions on Kubernetes resources.
   caveats: |
     Usage:
-      kubectl bulk-action <get resourceTypes> (get|list|create|update|delete|remove|rollout) <command parameters>
+      kubectl bulk-action <get resourceTypes> (get|list|create|update|delete|remove|rollout|scale) <command parameters>
+
+    Help:
+      kubectl bulk-action help        
 
     This plugin needs the following programs:
     * grep
     * sed
```

Comment:
I think it's safe to assume that both programs are available on both supported platforms. So feel free to remove this from your `caveats` section.

A valid caveat would be, if you require the GNU version of these programs.

Reply:
It seems meaningful, I removed it.